### PR TITLE
chore: conditionally hide offer note button for inquiry orders

### PR DESCRIPTION
**Pull request by xtina-starr**
Addresses: [PURCHASE-2393](https://artsyproduct.atlassian.net/browse/PURCHASE-2393)

The inquiry checkout flow repurposes this make offer flow as a [webflow in eigen](https://github.com/artsy/eigen/blob/master/src/lib/Scenes/Inbox/Screens/Checkout.tsx). Because an order initiated from an inquiry already has a message/note and the note field in the offer flow is redundant. This PR hides the offer note button for inquiry orders i.e. `isInquiryOrder: true`.

**Regular Order**
Screen Shot 2021-02-23 at 4 51 46 PM

**Inquiry Order**
Screen Shot 2021-02-23 at 4 25 35 PM

cc @artsy/purchase-devs

### CODE REVIEW

**Comment by xtina-starr** [comment]
Anyone have a sense for what's going on with the following error? Doesn't seem related to the changes in this PR.

Screen Shot 2021-02-23 at 5 53 38 PM

**Comment by starsirius** [comment]
This might be because it's submitted from you own fork. Try pushing it to the artsy repo and creating a new PR.

File: src/v2/Apps/Order/Routes/Respond/index.tsx [hunk 1]
```diff
@@ -277,20 +277,24 @@ export class RespondRoute extends Component<RespondProps, RespondState> {
                         onChange={offerValue => this.setState({ offerValue })}
                         onFocus={this.onOfferInputFocus.bind(this)}
                       />
-                      <Spacer mb={0.5} />
-                      <RevealButton
-                        align="left"
-                        buttonLabel="Add note to seller"
```
Comment (starsirius) [inline src/v2/Apps/Order/Routes/Respond/index.tsx:283]: Realized when collectors submit a counteroffer, they can also include a note. I think it still makes sense but I'm curious about user's feedback.

Comment (xtina-starr) [reply src/v2/Apps/Order/Routes/Respond/index.tsx:283]: Still makes sense to show this offer note for the response view? And keep it hidden for the initial offer view?

Comment (starsirius) [reply src/v2/Apps/Order/Routes/Respond/index.tsx:283]: Oh sorry I meant still makes sense to hide it. 😅

**Review by starsirius — APPROVED** [review]
A [new build](https://app.circleci.com/pipelines/github/artsy/force?branch=hide-note-inquiry-order) is pushed to the artsy repo and managed to update this PR that's originally submitted from your fork. (I was under the impression that you have to open a new PR.) In any way, it passed now! 🎉

File: src/v2/Apps/Order/Routes/Respond/index.tsx (hunk 1 above)
Comment (xtina-starr) [reply src/v2/Apps/Order/Routes/Respond/index.tsx:283]: Ah yeah I understand now.

**Pull request merged**